suitcase/tiff_stack/__init__.py

```diff
@@ -166,10 +167,8 @@ class Serializer(event_model.DocumentRouter):
     def __init__(self, directory, file_prefix='{uid}-', bigtiff=False,
                  byteorder=None, imagej=False, **kwargs):
 
-        if isinstance(directory, (str, Path)):
-            self._manager = suitcase.utils.MultiFileManager(directory)
-        else:
-            self._manager = directory
+        self._directory = directory
+        self._templated_directory = directory
```

Comment:
I like this initialization. I think it should be done the same way for the `tiff_series` as well.

Reply:
will do, it was an oversight